### PR TITLE
Update changelog to have 2020.1 and 2020.2 version

### PR DESCRIPTION
**Pull request by sebastienlagarde**
all in title

### CODE REVIEW

File: com.unity.render-pipelines.core/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [11.0.0] - 2020-10-21
+## [2022.2.0] - 2021-11-1
```
Comment (theo-pnv) [inline com.unity.render-pipelines.core/CHANGELOG.md:7]: Please add a `0` when it's a single-digit date, we had issues with validation tests in the past.
Example: `2021-11-01`

Comment (theo-pnv) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: Like in this test https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/srp-bumpversion/.yamato%252F_projectcontext.yml%2523test_all_project_windows_trunk/4831948/job

Comment (theo-pnv) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: Note: For all packages

File: com.unity.render-pipelines.high-definition-config/CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [11.0.0] - 2020-10-21
+## [2022.2.0] - 2021-11-1
```
Comment (theo-pnv) [inline com.unity.render-pipelines.high-definition-config/CHANGELOG.md:7]: Shouldn't it be 2021.2.0 instead? 
Note: For all packages

**Review by theo-pnv — CHANGES_REQUESTED** [review]

**Review by stramit — COMMENTED** [review]
I believe that because we are no longer doing package releases we do NOT need the date. It would be good to remove that.

**Comment by sebastienlagarde** [comment]
> I believe that because we are no longer doing package releases we do NOT need the date. It would be good to remove that.

But I guess package validation suite if we don't have a date...

File: com.unity.render-pipelines.high-definition-config/CHANGELOG.md (hunk 2 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition-config/CHANGELOG.md:7]: good catch!

File: com.unity.render-pipelines.core/CHANGELOG.md (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: good point

**Comment by theo-pnv** [comment]
I don't know if we would still be respecting the KeepAChangelog format if we removed the dates. https://keepachangelog.com/en/1.0.0/
image

**Review by theo-pnv — APPROVED** [review]

**Review by phi-lira — APPROVED** [review]

**Pull request merged**